wordcount_and_median.py

```diff
@@ -1,41 +1,40 @@
 import os
-from collections import Counter
 import re
+import csv
+import sys
 import bisect
-from statistics import median
-
+import statistics
+from collections import Counter
```

Comment:
I did two things here with the imports:

1) I re-ordered them to be in ascending order of line length.  This is purely a ~~stylistic~~ ~~OCD~~ personal preference and not like a universal Python convention or anything.

2) Below you were raising ``StatisticsError`` but this wasn't being imported.  So I changed it to raise ``statistics.StatisticsError`` instead and then changed this import.  Ironically, without this fix we could potentially raise an exception while trying to raise an exception (yo dawg).